crates/primitives/src/transaction/mod.rs

```diff
@@ -1587,6 +1587,69 @@ impl Decodable2718 for TransactionSigned {
     }
 }
 
+// todo: remove when merged <https://github.com/alloy-rs/alloy/pull/1412>
+impl alloy_consensus::Transaction for TransactionSigned {
```

Comment:
can we submit this seperately?
this todo can be removed because we decided against autoimpl on deref